### DISCUSSION TITLE
v5.90.3 pollution global variables jquery $

### DISCUSSION THREAD

**Discussion by qiujunbiao** [webpack/webpack#18162] [Q&A]
v5.90.3
<img width="1100" alt="image" src="https://github.com/webpack/webpack/assets/3970608/03189902-0d9b-4f1e-862b-24ddb902e27b">

v5.89.0
<img width="1023" alt="image" src="https://github.com/webpack/webpack/assets/3970608/7e6633e0-4d4d-4ebc-8300-d65f944c770c">

Causing variables in jquery to be overwritten

**Comment by alexander-akait** [comment]
It is not webpack code, if you need a help, please create reproducible test repo
